lib/container.js

```diff
@@ -135,6 +151,14 @@ export default class Container {
     });
   }
 
+  _createProfile(user, profile) {
+    let record = new this.UserRecord({
+      _id: 'user/' + user.id,
+      ...profile
+    });
+    return this.publicDB.save(record);
+  }
```

Comment:
`saveUser` accepting user object which is intended for update of username, email and user role.